common/analysis/command_file.lex

```diff
@@ -51,10 +51,13 @@ FlagWithArg {ParamPrefix}{Name}=
 Arg {Value}
 Param {Value}
 
+CommentPrefix #|#{Space}
+Comment {CommentPrefix}{Value}
```

Comment:
Recommend formulating this as a a 'start' and 'rest-of-line', e.g.

https://github.com/google/verible/blob/d811cea2c593985cabbd043d338daa4a8ed08493/verilog/parser/verilog.lex#L175

'Value' in this case should also allow for spaces and tabs (which isn't in 'AnyChar', above).

And suggest naming this `EOLComment`.

I see that the newline is excluded from the comment token and handled separately, that's ok.